### PR TITLE
Overriding label_from_instance function of ModelChoiceField

### PR DESCRIPTION
**Pull request by sanjanavsdvn**

### CODE REVIEW

**Comment by sanjanavsdvn** [comment]
@erickpeirson I checked the html files. We are not referencing any variable directly. We are referring to the fields only. (name, type, url as such)

File: cookies/forms.py [hunk 1]
```diff
@@ -17,6 +17,10 @@
 from cookies import authorization
 
 
+class CustomModelChoiceField(forms.ModelChoiceField):
+    def label_from_instance(self, obj):
+         return obj.name
+
```
Comment (erickpeirson) [inline cookies/forms.py]: Always leave two blank lines above any top-level class or function declaration.

**Comment by erickpeirson** [comment]
@sanjanavsdvn This looks good with respect to form fields. To complete the story we will also need to correct the problems in the templates; would you like to create a separate PR for that, or shall we keep this open until you're done?

**Comment by sanjanavsdvn** [comment]
@erickpeirson Check my previous commit. I took a look at the templates folder. There were no instances where only the model was referred as such. Fields were written explicitly.

**Comment by erickpeirson** [comment]
@sanjanavsdvn Here's one: https://github.com/diging/amphora/blob/master/cookies/templates/entity_details.html#L79 (``{{ relation.target|.... }}``

To find these you'll have to actually browse around the user interface.

**Comment by sanjanavsdvn** [comment]
@erickpeirson Oh. I didn't understand that. I am only familiar with resources and collections right now. alright. I'll check. Let this be open till I resolve all that.

**Comment by sanjanavsdvn** [comment]
@erickpeirson Kindly take a look

**Comment by erickpeirson** [comment]
@sanjanavsdvn Fantastic! Thank you

**Pull request merged**